### PR TITLE
Add search bar

### PR DESCRIPTION
**Pull request by drewbutlerbb4**
Related: #166 

Adds a search bar to the asset featured page.

Signed-off-by: Andrew-Butler <Andrew.Butler@ibm.com>

### CODE REVIEW

**Comment by ckadner** [comment]
@drewbutlerbb4 can you add a screenshot with where you moved the search box?

**Comment by drewbutlerbb4** [comment]
@ckadner I've left it in place for now. I can move it when we have a better idea of what we want. Here is where it stands now.

Screen Shot 2021-07-30 at 10 51 09 AM

File: dashboard/origin-mlx/src/styles/Models.css [hunk 1]
```diff
@@ -367,6 +367,16 @@ td>span.checked {
   overflow-x: hidden;
 }
 
+<<<<<<< HEAD
```
Comment (Tomcli) [inline dashboard/origin-mlx/src/styles/Models.css:370]: Need to fix this conflict

**Comment by Tomcli** [comment]
in the package.json, we need to add the `"fuzzysort": "^1.1.4",` package

File: dashboard/origin-mlx/src/styles/Models.css (hunk 1 above)
Comment (drewbutlerbb4) [reply dashboard/origin-mlx/src/styles/Models.css]: Thanks @Tomcli

**Pull request merged**